bulk/cli/image.py

```diff
@@ -17,10 +17,17 @@
 
 from bulk._bokeh_utils import get_color_mapping
 
+import urllib3
+
 
 def encode_image(path):
-    with open(path, "rb") as image_file:
-        enc_str = base64.b64encode(image_file.read()).decode("utf-8")
+    if type(path) == str and path.startswith('http'):
+        http = urllib3.PoolManager()
+        contents = http.request('GET', path)
+        enc_str = base64.b64encode(contents.data).decode("utf-8")
+    else:
+        with open(path, "rb") as image_file:
+            enc_str = base64.b64encode(image_file.read()).decode("utf-8")
```

Comment:
Wouldn't it be a lot less computationally intensive to have the frontend do this fetching? Something like:

```python
if path.startswith('http'):
     return f'<img style="object-fit: scale-down;" width="100%" height="100%" src="{path}">'
```